src/registrationSystem/utils.py

```diff
@@ -41,7 +41,7 @@ def send_email(user):
 
     # The connector binds the randomized token to the RaffleEntry
     # from which the account information will be retreived.
-    connector = EmailConfirmation.objects.create(raffleEntryId=user)
+    connector = EmailConfirmation.objects.get_or_create(raffleEntryId=user)
```

Comment:
get_or_create returns two things which you need to take care of
```suggestion
    connector, _ = EmailConfirmation.objects.get_or_create(raffleEntryId=user)
```